scripts/dump_schema.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Later we can consider moving all the db related scripts under `scripts/db/` or similar. There are already a lot of them.